modules/caddytls/connpolicy.go

```diff
@@ -537,7 +537,8 @@ func setDefaultTLSParams(cfg *tls.Config) {
 
 // LeafCertClientAuth verifies the client's leaf certificate.
 type LeafCertClientAuth struct {
-	TrustedLeafCerts []*x509.Certificate
+	LeafCertificateLoadersRaw []json.RawMessage `json:"leaf_certs_loaders,omitempty" caddy:"namespace=tls.leaf_cert_loader inline_key=loader"`
+	TrustedLeafCerts          []*x509.Certificate
```

Comment:
Now that I see this I wonder if this field should be unexported. (Or be omitted from JSON:
```
`json:"-"`
```
)

Reply:
Can I ask how will we be providing the loaders then if we are not including them in the JSON?  Not questioning your comment but rather trying to wrap my head around the entire process that would get the certificates to the `LeafCertClientAuth` . Thank you!

Reply:
Sorry, to clarify: `TrustedLeafCerts` does not currently have a JSON tag. It's exported though, which means users can configure it like `TrustedLeafCerts: ...` in their JSON config, which isn't conventional, as we use `snake_casing`: `trusted_leaf_certs`. If the user is intended to use the `LeafCertificateLoadersRaw` field to set `TrustedLeafCerts` and this field isn't intended to be user-configurable directly, it should be unexported.

Reply:
Oh that's completely my bad. I thought you were talking about the `LeafCertificateLoadersRaw` field. Sorry. This should be addressed now.